apps/contributor/models.py

```diff
@@ -88,7 +96,7 @@ class ContributorTeam(ArchivableResource, UserResource, FirebasePushResource):
 
     @typing.override
     def __str__(self):
-        return self.name
+        return f"Archived - {self.name}" if self.is_archived else self.name
```

Comment:
@thenav56 Is this where we update the label? Or, we just want to update the label on the AdminPanel?

Reply:
@thenav56 this is fine.